src/components/Toast/Toast.css

```diff
@@ -0,0 +1,36 @@
+.toast {
+  align-items: center;
+  display: flex;
+  position: fixed;
+  bottom: 24px;
+  right: 24px;
+  background-color: #08d69f;
+  width: auto;
+  height: 56px;
+  border-radius: 5px;
+}
+
+.toast__icon {
+  margin-left: 8px;
+}
+
+.toast__button {
+  position: absolute;
+  transform: translate(-10%, -90%);
+  top: 42%;
+  left: 94%;
+  background-color: transparent;
+  color: white;
+}
+
+.toast__text {
+  margin: 8px;
+  padding-right: 80px;
```

Comment:
this should be inside the `.toast` class. I mean, it works one way or the other, but it would be more conceptually correct if all the toast was 80px wider instead of only the text